### ISSUE TITLE
SYSLIB0051 Deprecation error

### ISSUE THREAD

**Issue by joelwanner** [OneIdentity/safeguard-ps#505]
I'm getting the following error on PowerShell 7.4.0 and the most recent version of `safeguard-ps` @[89ccddd](https://github.com/OneIdentity/safeguard-ps/commit/89ccddd5b58e63bc64d57433d772c301d5c431d0):

```powershell
PowerShell 7.4.0
PS C:\Users\jowa> Import-Module -Name \\fs-homez\jowa$\Documents \PowerShel]\Modules\safeguard-ps\
PS C:\Users\jowa> Connect-Safeguard spp-prod local superadmin
Password: *****************
Login Successful.
PS C:\Users\jowa> Get-SafeguardUser admin
Add-Type: (30,13): error SYSLIB0051: "Exception.Exception(SerializationInfo, StreamingContext)" ist veraltet: "This API
supports obsolete formatter-based serialization. It should not be called or extended by application code."
(https://aka.ms/dotnet-warnings/SYSLIB0051)                      : base(info, context) {}             ^
PS C:\Users\jowa>
```

How can I resolve this issue?

Cheers,
Joel

**Comment by jasonlarke2023** [comment]
I'm also having this problem. Have you identified any sort of solution?

**Comment by azirish1998** [comment]
+1 on this one
Rolling back to PS 7.3.x resolved the issue.  possibly an incompatibility with new .net release?

**Comment by petrsnd** [comment]
Should be fixed by #509 
New prerelease build is: https://www.powershellgallery.com/packages/safeguard-ps/7.5.135229-pre
Working on an update for 7.4

**Comment by petrsnd** [comment]
Current release now has the fix: https://www.powershellgallery.com/packages/safeguard-ps/7.4.135231

@joelwanner @azirish1998 @jasonlarke2023 
Can you please verify this is working for you?  Please reopen the issue if there is a problem
